src/modules/v1/users/users.service.ts

```diff
@@ -37,10 +46,48 @@ export default class UsersService {
       createdUserAndDriver = { user: createdUser, driver };
     });
     await session.endSession();
-    if (createdUserAndDriver) return createdUserAndDriver;
+    const otp = generateOtp(6);
```

Comment:
All these can go into one function, when you are reading code... you want to make sure its as easy as possible to read.

Reply:
Yes boss.